benchopt/utils/slurm_executor.py

```diff
@@ -22,6 +23,13 @@ def get_slurm_launch():
     return _LAUNCHING_SLURM
 
 
+def get_slurm_folder(benchmark, job_name="benchopt_run"):
```

Comment:
Could you make the `job_name` value a constant `SLURM_JOB_NAME` so that it is consistent in `get_slurm_folder` and `get_slurm_executor`.
Also, I think you can remove the parameter `job_name` as it cannot be be passed anywhere I guess.

Reply:
Everything has been addressed 😇